services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationEntity.kt

```diff
@@ -24,7 +26,16 @@ data class ApplicationEntity(
     var userId: String?,
     @Enumerated(EnumType.STRING) val applicationType: ApplicationType,
     @Type(type = "json") @Column(columnDefinition = "jsonb") var applicationData: ApplicationData,
+    @ManyToOne @JoinColumn(updatable = false, nullable = false) var hanke: HankeEntity?,
```

Comment:
This says nullable=false, but the foreign key column doesn't have NOT NULL. This is also a nullable type, so should it be nullable or not?

Reply:
Not nullable. Added the missing constraint.